src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2121,263 @@ definitions:
         description: path to patch
       value:
         type: string
-        description: value of path
\ No newline at end of file
+        description: value of path
+  
+  subscribe_user_to_alert_body:
+    type: object
+    description: The definition for subscribing emailId's to query or schedule IDs for a given alertType
```

Comment:
"The definition"? 

Pay attention to where these strings are appearing in the swagger interface and if they will make sense to the person reading them in that context. "Definition" means something to somebody constructing the swagger, but will not make sense outside of that.